include/jemalloc/internal/jemalloc_internal_defs.h.in

```diff
@@ -309,6 +309,17 @@
  */
 #undef JEMALLOC_THP
 
+/* Defined if posix_madvise is available. */
+#undef JEMALLOC_HAVE_POSIX_MADVISE
+
+/*
+ * Method for purging unused pages using posix_madvise.
+ *
+ *   posix_madvise(..., POSIX_MADV_DONTNEED)
+ */
+#undef JEMALLOC_PURGE_POSIX_MADVISE_DONTNEED
+#undef JEMALLOC_PURGE_POSIX_MADVISE_DONTNEED_ZEROS
```

Comment:
It looks like this never gets set?

Reply:
Right, it's not set. I added this just to mirror JEMALLOC_PURGE_MADVISE_DONTNEED.